From 9d131438bcb022617c95e736cfe260660bf8cacb Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Fri, 10 Feb 2012 10:39:37 +0800
Subject: [PATCH 1001/2508] ENGR00174128-3 Revert "Remove the discharge for
 VBUS and DP -3"

This reverts commit cc9616316bf044382ec422da5a6d4ed007235a3d.

Signed-off-by: Peter Chen <peter.chen@freescale.com>
---
 include/linux/fsl_devices.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/linux/fsl_devices.h b/include/linux/fsl_devices.h
index cc243ba..e96cca4 100644
--- a/include/linux/fsl_devices.h
+++ b/include/linux/fsl_devices.h
@@ -124,6 +124,7 @@ struct fsl_usb2_platform_data {
 	void (*wakeup_handler)(struct fsl_usb2_platform_data *);
 	void (*hsic_post_ops)(void);
 	void (*hsic_device_connected)(void);
+	void (*gadget_discharge_dp) (bool);
 
 	struct fsl_usb2_wakeup_platform_data *wakeup_pdata;
 	struct platform_device *pdev;
-- 
1.7.7.4

